plot_continuous_monitor.py

```diff
@@ -129,106 +94,66 @@ def plot_continuous_monitor(filename, open_graphs=False):
       free_off_heap_memory = json_data["Free Off-Heap Memory Bytes"]
 
     data = [
-      time - start,
-      xvdf_total_utilization,
-      xvdb_total_utilization,
-      cpu_total / 8.0,
-      bytes_received / 125000000.,
-      bytes_transmitted / 125000000.,
-      running_compute_monotasks,
-      running_macrotasks,
-      gc_fraction,
-      outstanding_network_bytes / (1024 * 1024),
-      macrotasks_in_network,
-      macrotasks_in_compute,
-      cpu_system / 8.0,
-      macrotasks_in_disk,
-      xvdf_read_throughput,
-      xvdf_write_throughput,
-      xvdb_read_throughput,
-      xvdb_write_throughput,
-      xvdf_running_disk_monotasks,
-      xvdb_running_disk_monotasks,
-      free_heap_memory / BYTES_PER_GIGABYTE,
-      free_off_heap_memory / BYTES_PER_GIGABYTE,
-      local_running_macrotasks]
-    write_data(out_file, data)
-  out_file.close()
-
-  # Write plot files.
-  utilization_plot_filename = "%s_utilization.gp" % filename
-  utilization_plot_file = open(utilization_plot_filename, "w")
-  for line in open(os.path.join(scripts_dir, "plot_utilization_base.gp"), "r"):
-    new_line = line.replace("__NAME__", out_filename)
-    utilization_plot_file.write(new_line)
-  utilization_plot_file.close()
+      ('time', time - start),
+      ('xvdf utilization', xvdf_total_utilization),
+      ('xvdb utilization', xvdb_total_utilization),
+      ('cpu utilization', cpu_total / 8.0),  # Cores in machine?
+      ('bytes received', bytes_received / 125000000.),
+      ('bytes transmitted', bytes_transmitted / 125000000.),  # what is the 125000000 magic number?
```

Comment:
125000000 is the number of bytes in a gigabit (these scripts assume the machines have 1Gbps network links)